zellij-utils/src/input/mod.rs

```diff
@@ -41,6 +41,7 @@ pub fn get_mode_info(
             keybinds.push(("x".to_string(), "Close".to_string()));
             keybinds.push(("r".to_string(), "Rename".to_string()));
             keybinds.push(("s".to_string(), "Sync".to_string()));
+            keybinds.push(("Tab".to_string(), "Last".to_string()));
```

Comment:
Same as above, I think `Toggle` might carry the intention better than `Last`.

Reply:
Done.